commit 2c9bdf6e4771a5966a4f0d6bea45a1c7f38312d7
Author: Gustavo A. R. Silva <gustavoars@kernel.org>
Date:   Fri May 28 15:03:35 2021 -0500

    floppy: Fix fall-through warning for Clang
    
    In preparation to enable -Wimplicit-fallthrough for Clang, fix a warning
    by explicitly adding a break statement instead of letting the code fall
    through to the next case.
    
    Link: https://github.com/KSPP/linux/issues/115
    Link: https://lore.kernel.org/linux-hardening/47bcd36a-6524-348b-e802-0691d1b3c429@kernel.dk/
    Suggested-by: Jens Axboe <axboe@kernel.dk>
    Signed-off-by: Gustavo A. R. Silva <gustavoars@kernel.org>
    Signed-off-by: Denis Efremov <efremov@linux.com>

diff --git a/drivers/block/floppy.c b/drivers/block/floppy.c
index e96ad5b2c35e..cde70b0a55dd 100644
--- a/drivers/block/floppy.c
+++ b/drivers/block/floppy.c
@@ -2123,6 +2123,7 @@ static void format_interrupt(void)
 	switch (interpret_errors()) {
 	case 1:
 		cont->error();
+		break;
 	case 2:
 		break;
 	case 0: